Mathlib/NumberTheory/NumberField/InfinitePlace/TotallyRealComplex.lean

```diff
@@ -139,26 +150,48 @@ theorem IsTotallyReal.le_maximalRealSubfield (E : Subfield K) [IsTotallyReal E]
   refine RingHom.congr_fun ?_ _
   exact ComplexEmbedding.isReal_iff.mp <| isReal_mk_iff.mp <| isReal _
 
-theorem isTotallyReal_iff_le_maximalRealSubfield {E : Subfield K} :
-    IsTotallyReal E ↔ E ≤ maximalRealSubfield K :=
-  ⟨fun h ↦ h.le_maximalRealSubfield, fun h ↦ IsTotallyReal.ofRingEquiv
-    (RingEquiv.ofBijective _ (Subfield.inclusion h).rangeRestrictField_bijective).symm⟩
-
-instance isTotallyReal_sup {E F : Subfield K} [IsTotallyReal E] [IsTotallyReal F] :
+theorem isTotallyReal_iff_le_maximalRealSubfield {E : Subfield K} [Algebra.IsIntegral E K] :
```

Comment:
Those are still in the `NumberField` namespace, right? I think this is fine, as it is the main application, just to check if we need deprecations.